project-base/storefront/components/Blocks/Product/LastVisitedProducts/LastVisitedProducts.tsx

```diff
@@ -0,0 +1,49 @@
+import { ProductsSlider } from 'components/Blocks/Product/ProductsSlider';
```

Comment:
The app is broken because of this feature ❌

![image](https://github.com/shopsys/shopsys/assets/11465991/464e1fb8-97b4-4e12-8c6f-e634e1fb8668)

Reply:
Fixed